src/sidemenu/BoardsMenuSection.tsx

```diff
@@ -51,14 +51,15 @@ const BoardsMenuSection: React.FC<BoardsMenuSectionProps> = (props) => {
           ))}
         </div>
         <div
+          aria-busy="true"
           className={classnames("boardSection-board-items", {
             visible: loading,
             hidden: isEmpty,
           })}
           key="loading-items-section"
         >
           {Array.from({ length: placeholdersCount || 0 }).map((_, index) => (
-            <div className="boardItem" key={`board-item-${index}`}>
+            <div className="boardItem" aria-label="loading board link" key={`board-item-${index}`}>
```

Comment:
Torn about this aria-label. What will happen this way is the screen reader will read "loading board link" `placeholdersCount` times. That doesn't sound like an ideal experience. 

I don't have enough accessibility knowledge to know if there is an even better way to do this, but I would move this area-label on the container where you added `aria-busy="true"` and says something like "Placeholder for loading boards".

There is a question of how to test these links without the aria-label, but I'll think about it once I review that section.

Reply:
Addressed below